### ISSUE TITLE
psfmakefea should sort classes when it can

### ISSUE THREAD

**Issue by bobh0303** [silnrsi/pysilfont#62]
In order to store feature.fea in each source UFO, we are now using `psfmakefea` in preflight. Every time the UFO is touched, the classes built by psfmakefea come out in a different order, which is thrashing the git file history for the generated fea. 

I propose that classes created due to glyph name extensions and due to anchors should be generated in a consistent order.

I _think_ the following will accomplish the desired goal without breaking other things:

```
    def make_classes(self) :
        for name, g in self.glyphs.items() :
```
should be changed to process self.glyphs.items() in order sorted by its keys.

Also, in `append_positions()`, the outer loop:
```
        for ap_nm, glyphs_w_ap in self.all_aps.items() :
```
should processed in sort order by AP names, and in the inner loop:
```
            for g in glyphs_w_ap :
```
should be processed in sort order by glyph name. 

There may be other loops that need to be sorted... I would need to understand the code better to know.

(Note that classes generated from a _classfile_ should not be sorted.)

**Comment by bobh0303** [comment]
`collections.OrderedDict` might be useful.

**Comment by DavidRaymond** [comment]
This is also an issue with regression testing, since Python 2 and Python 3 produce output in different orders.

**Comment by bobh0303** [comment]
This may have been fixed by commits e5e13157cf13cf6d0b8aa2a003df9464f34cfed1 and 6937fadda56bbab87a437f459400d006cafe9bf5 ... need to test further.
